pandas/tests/io/test_parquet.py

```diff
@@ -736,6 +737,10 @@ def test_unsupported_float16_cleanup(self, pa, path_type):
         data = np.arange(2, 10, dtype=np.float16)
         df = pd.DataFrame(data=data, columns=["fp16"])
 
+        # Cleanup fails in windows
+        if is_platform_windows():
```

Comment:
pls use the decorator form & put a meanigful error message

Reply:
I have added the skip condition in the pytest decorator and added a message now